client/src/test/utils/utils.test.js

```diff
@@ -0,0 +1,55 @@
+import { getFullname, makeQueryStringTransformable } from '../../utils/utils';
+
+it('Shoud check if getFullname util works', () => {
+  expect(getFullname({ firstName: 'Jack', lastName: 'John' })).toEqual(
```

Comment:
Is in it good to have file that contain data-> user1: {firstName: "abc",lastName:"xyz", fullName: "abc xyz"}?